src/System.Runtime/tests/System/TupleTests.cs

```diff
@@ -395,56 +430,64 @@ public static void Equals_GetHashCode()
             tupleDriverA = new TupleTestDriver<short, int, long, string, Char, Single, Double, DateTime, Tuple<bool, object>, TimeSpan>((short)10000, (int)1000000, (long)10000000, "2008?7?2?", '0', (Single)0.0001, (Double)0.0000001, now, Tuple.Create(false, myObj), TimeSpan.Zero);
             tupleDriverB = new TupleTestDriver<short, int, long, string, Char, Single, Double, DateTime, Tuple<bool, object>, TimeSpan>((short)10000, (int)1000000, (long)10000000, "2008?7?2?", '0', (Single)0.0001, (Double)0.0000001, now, Tuple.Create(false, myObj), TimeSpan.Zero);
             tupleDriverC = new TupleTestDriver<short, int, long, string, Char, Single, Double, DateTime, Tuple<bool, object>, TimeSpan>((short)10001, (int)1000001, (long)10000001, "2008?7?3?", '1', (Single)0.0002, (Double)0.0000002, now.AddMilliseconds(1), Tuple.Create(true, myObj), TimeSpan.MaxValue);
-            tupleDriverA.Equals_GetHashCode(tupleDriverB, true, true);
-            tupleDriverA.Equals_GetHashCode(tupleDriverC, false, false);
+            tupleDriverA.Equals_GetHashCode(tupleDriverB, true, true, netfx);
+            tupleDriverA.Equals_GetHashCode(tupleDriverC, false, false, netfx);
         }
 
+
         [Fact]
-        public static void CompareTo()
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, ".NET Framework hasn't received the fix for https://github.com/dotnet/corefx/issues/18528 yet.")]
+        public static void CompareTo() => CompareTo(netfx: false);
+
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework, ".NET Framework hasn't received the fix for https://github.com/dotnet/corefx/issues/18528 yet.")]
+        public static void CompareTo_netfx() => CompareTo(netfx: true);
```

Comment:
Nit: you can use PlatformDetection.IsFullFramework to avoid having to explicitly have a `netfx` parameter